Formula/op25.rb

```diff
@@ -0,0 +1,64 @@
+class Op25 < Formula
+  desc "Software-defined analyzer for APCO P25 signals"
+  homepage "https://osmocom.org/projects/op25/wiki"
+  url "git://op25.osmocom.org/op25.git"
+
+  depends_on "cmake" => :build
+  depends_on "cppunit" => :build
+  depends_on "swig" => :build
+  depends_on "gnuradio"
+  depends_on "gr-osmosdr"
```

Comment:
Successfully builds against `gr-osmosdr@0.1.4`, will need to be updated if/when https://github.com/Homebrew/homebrew-core/pull/50921 is unblocked and merged.